src/patternfly/components/Slider/examples/Slider.md

```diff
@@ -0,0 +1,267 @@
+---
+id: 'Slider'
+beta: true
+section: components
+cssPrefix: pf-c-slider
+---
+
+## Examples
+### Discrete
+```hbs
+{{#> slider
+  slider--value-min="0"
+  slider--value-max="8"
+  slider--value-now="5"
```

Comment:
why "now" instead of a word like "current"

Reply:
`aria-valuenow` - https://www.w3.org/TR/wai-aria-practices/examples/slider/slider-1.html#rps_label

Though it's just an internal handlebars var so we can call it whatever you'd like to change it.

Reply:
oh right, thats good with me didnt realize it was for aria